cores/esp32/MacAddress6.h

```diff
@@ -0,0 +1,63 @@
+//-----------------------------------------------------------------------------
+// MacAddress6.h - class to make it easier to handle BSSID and MAC addresses.
+// 
+// Copyright 2022 David McCurley
+// Licensed under the Apache License, Version 2.0 (the "License").
+// You may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+//     
+// Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+//   limitations under the License.
+//-----------------------------------------------------------------------------
+
+#ifndef MacAddress6_h
+#define MacAddress6_h
+
+#include <WString.h>
+
+// A class to make it easier to handle and pass around 6-byte BSSID and MAC addresses.
+class MacAddress6 {
```

Comment:
Maybe make it inherit from Printable?
class MacAddress6: public Printable {

Also add virtual size_t printTo(Print& p) const;

Reply:
I specifically did not implement this as I don't understand Printable.  Let's get this class added.  If someone else wants to implement Printable later, they are welcome to do so.

Reply:
I'll help on it.